web/src/components/tasks/LicenseTask.tsx

```diff
@@ -46,20 +45,23 @@ export const LicenseTask = (props: Props): ReactElement => {
 
   const licenseNameForTask = taskIdLicenseNameMapping[props.task.id];
   const hasCompletedSearch = !!licenseDetails?.lastUpdatedISO;
-  const searchHasError = !!licenseDetails?.hasError;
 
   const allFieldsHaveValues = (nameAndAddress: LicenseSearchNameAndAddress): boolean => {
     return !!(nameAndAddress.name && nameAndAddress.addressLine1 && nameAndAddress.zipCode);
   };
 
   useEffect(() => {
     if (!business) return;
-    const licenseDetailsReceived = business.licenseData?.licenses?.[licenseNameForTask]?.lastUpdatedISO;
```

Comment:
pebble - `hasDoneLicenseCheck` doesn't need to exist. you should consider the logic based on the 2by2 matrix exercise we did yesterday to figure out all the possible outcomes.